website/docs/r/iothub.html.markdown

```diff
@@ -133,6 +139,20 @@ A `route` block supports the following:
 
 * `enabled` - (Required) Used to specify whether a route is enabled.
 
+---
+
+A `fallback_route` block supports the following:
+
+* `name` - (Optional) The name of the route. The name can only include alphanumeric characters, periods, underscores, hyphens, has a maximum length of 64 characters, and must be unique.
```

Comment:
We typically don't list the validation requirements here and let the validation function do inform.